packages/ui/components/index.ts

```diff
@@ -1,8 +1,27 @@
-export * from "./avatar";
-export * from "./badge";
-export * from "./breadcrumb";
-export * from "./button";
-export * from "./buttonGroup";
-export * from "./form";
+export { Avatar, AvatarGroup } from "./avatar";
+export type { AvatarProps } from "./avatar";
+export { Badge } from "./badge";
+export type { BadgeProps } from "./badge";
+export { Breadcrumb, BreadcrumbContainer, BreadcrumbItem } from "./breadcrumb";
+export { Button } from "./button";
+export type { ButtonBaseProps, ButtonProps } from "./button";
+export { ButtonGroup } from "./buttonGroup";
+export {
+  Checkbox,
+  EmailField,
+  EmailInput,
+  FieldsetLegend,
+  Form,
+  HintsOrErrors,
+  Input,
+  InputField,
+  InputGroupBox,
+  InputLeading,
+  Label,
+  PasswordField,
+  TextArea,
+  TextAreaField,
+  TextField,
+} from "./form";
```

Comment:
I'd rather concentrate the complexity here instead of each individual file. If we stop exporting a file then we expect an error and will help us find old/deprecated code and replace It easily.